### PR TITLE
DOC: Improve appearance of README

### PR DESCRIPTION
**Pull request by bashtage**
Add additional badges and fix rst errors

[skip ci]

### CODE REVIEW

**Comment by bashtage** [comment]
Don't merge yet - the appveyor badge is incorrect.  Only the account holder can lookup the randomstring that is needed to get the right one.  It is in badges which I think is in your settings.

**Comment by bashtage** [comment]
You can see output 

https://github.com/bashtage/statsmodels/blob/beautify-readme/README.rst

File: README.rst [hunk 1]
```diff
@@ -120,10 +128,16 @@ Discussions take place on our mailing list.
...
 ===========
 
 Bug reports can be submitted to the issue tracker at
 
     https://github.com/statsmodels/statsmodels/issues
+
+.. |Travis Build Status| image:: https://travis-ci.org/statsmodels/statsmodels.svg?branch=master
+   :target: https://travis-ci.org/statsmodels/statsmodels
+.. |Appveyor Build Status| image:: https://ci.appveyor.com/api/projects/status/odc5c4ukhru5xicl/branch/master?svg=true
+   :target: https://ci.appveyor.com/project/josef-pkt/statsmodels/branch/master
```
Comment (josef-pkt) [inline README.rst]: SVG image URL for project status
`https://ci.appveyor.com/api/projects/status/gx18sd2wc63mfcuc?svg=true`

but there is also a badge for master branch status:
https://ci.appveyor.com/api/projects/status/gx18sd2wc63mfcuc/branch/master?svg=true
with link in markdown
[Build status](https://ci.appveyor.com/project/josef-pkt/statsmodels/branch/master)

**Comment by bashtage** [comment]
@josef-pkt Updated to correct appveyor.  Should be ready.

**Comment by josef-pkt** [comment]
(this will be essentially the last for 0.8)

As far as I can tell right now, this is also the basis for the description on pypi. So a few more changes to update this. 
(inline comments to come)

File: README.rst [hunk 2]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
...
-* glm: Generalized linear models with support for all of the one-parameter
+* Linear regression models:
+
+  - Ordinary least squares
+  - Generalized least squares
+  - Weighted least squares
+  - Least squares with autoregressive errors
+
+* GLM: Generalized linear models with support for all of the one-parameter
   exponential family distributions.
```
Comment (josef-pkt) [inline README.rst]: missing (after GLM)
`* GEE: Generalized Estimating Equations for one-way clustered or longitudinal data.`

File: README.rst [hunk 3]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
...
-* discrete: regression with discrete dependent variables, including Logit, Probit, MNLogit, Poisson, based on maximum likelihood estimators
+* Discrete model:
+
+  - Logit and Probit
+  - MNLogit - Multinomial logit
+  - Poisson regresion
+
```
Comment (josef-pkt) [inline README.rst]: missing
` - Negative Binomial`

File: README.rst [hunk 4]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
...
+  - Tools for reading Stata .dta files into numpy arrays.
+  - Table output to ascii, latex, and html
+
+* Miscellaneous models
+* Sandbox: statsmodels contains a sandbox folder with code in various stages of
+  developement and testing which is not considered "production ready".   This covers
+  among others
+
+  - Mixed (repeated measures) Models
+  - GARCH models
```
Comment (josef-pkt) [inline README.rst]: delete Mixed and GARCH (obsolete)

File: README.rst [hunk 5]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
...
+  - Generalized method of moments (GMM) estimators
+  - Kernel regression
+  - Various extensions to scipy.stats.distributions
+  - Panel data models
+  - Generalized additive models
```
Comment (josef-pkt) [inline README.rst]: delete `- Generalized additive models`

File: README.rst [hunk 6]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
...
-* linear regression models: Generalized least squares (including weighted least squares and
-  least squares with autoregressive errors), ordinary least squares.
-* glm: Generalized linear models with support for all of the one-parameter
+* Linear regression models:
+
+  - Ordinary least squares
+  - Generalized least squares
+  - Weighted least squares
+  - Least squares with autoregressive errors
+
```
Comment (josef-pkt) [inline README.rst]: missing (I think we add on top "star" level)

`* Mixed Linear Model with mixed effects and variance components`

File: README.rst [hunk 7]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
...
-* sandbox: statsmodels contains a sandbox folder with code in various stages of
-  developement and testing which is not considered "production ready".
-  This covers among others Mixed (repeated measures) Models, GARCH models, general method
-  of moments (GMM) estimators, kernel regression, various extensions to scipy.stats.distributions,
-  panel data models, generalized additive models and information theoretic measures.
-
 
-Where to get it
-===============
+* Principal Component Analysis
```
Comment (josef-pkt) [inline README.rst]: I think before PCA

`* Imputation with MICE and regression on order statistic`

File: README.rst [hunk 8]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
 
    http://statsmodels.github.io/dev/index.html
```
Comment (josef-pkt) [inline README.rst]: we should replace this, or add `www.statsmodels.org/dev/index.html`

"www" is required, doesn't work without

File: README.rst [hunk 9]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
...
+  - Complete StateSpace modeling framework
+
+    - Seasonal ARIMA and ARIMAX models
+    - VARMA and VARMAX models
+    - Dynamic Factor models
+
+  - Markov switching models (MSAR), also known as Hidden Markov Models (HMM)
+  - Univariate time series analysis: AR, ARIMA
+  - Vector autoregressive models, VAR and structural VAR
+  - Descriptive statistics and process models for time series analysis
```
Comment (josef-pkt) [inline README.rst]: should also add
`- hypothesis tests for time series: unit root, cointegration and others`

File: README.rst [hunk 10]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
...
 
-* linear regression models: Generalized least squares (including weighted least squares and
-  least squares with autoregressive errors), ordinary least squares.
-* glm: Generalized linear models with support for all of the one-parameter
+* Linear regression models:
+
+  - Ordinary least squares
+  - Generalized least squares
+  - Weighted least squares
+  - Least squares with autoregressive errors
```
Comment (josef-pkt) [inline README.rst]: Quantile regression is missing, I'm not sure where it should go but we can put it under Linear regression models
`- Quantile regression`

File: README.rst [hunk 11]
```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
 
    http://statsmodels.github.io/dev/index.html
 
-
-This is currently a temporary solution that will be transformed into the new permanent location.
+**Note:** This is currently a temporary solution that will be transformed into the new permanent location.
```
Comment (josef-pkt) [inline README.rst]: delete note: this is no permanent, especially `www.statsmodels.org` can always go with any location change

**Comment by josef-pkt** [comment]
The "documentation" section is duplicate, once on top and once in the middle

File: README.rst [hunk 12]
```diff
@@ -101,6 +123,10 @@ The official documentation is hosted on SourceForge
 
     http://statsmodels.sf.net/
 
+or Github
+
+    http://statsmodels.github.io/dev/
+
 
 Windows Help
 ============
```
Comment (josef-pkt) [inline README.rst]: delete this entire `Windows Help` section. I haven't updated and used it in a long time.

**Comment by josef-pkt** [comment]
That's all I can see reading through it. Needs to go into 0.8 so it can be used for pypi

@bashtage  Can you update this within the next hour or so, otherwise I'll merge this and make the changes in a separate PR.

**Comment by bashtage** [comment]
Updated.

**Comment by josef-pkt** [comment]
great, looking good, and merging

**Pull request merged**